### ISSUE TITLE
Fix support for Python 3.8

### ISSUE THREAD

**Issue by fkromer** [quantumblacklabs/kedro-docker#30]
## Description

Installing `kedro-docker` under Python 3.8 is not possible.

## Context

[Python 3.8 support of `kedro` has been added recently.](https://github.com/quantumblacklabs/kedro/issues/124) `kedro-docker` does not support Python 3.8 yet: [`python_requires=">=3.6, <3.8",`](https://github.com/quantumblacklabs/kedro-docker/blob/develop/setup.py#L66)

## Steps to Reproduce

1. Install Python 3.8
2. Install kedro
3. Try to install kedro-docker

## Expected Result

Successful installation of `kedro-docker`.

## Actual Result

Installation aborts.

```
$ kedro --version
kedro, version 0.16.1
$ pip3 install --user kedro-docker
ERROR: Could not find a version that satisfies the requirement kedro-docker (from versions: none)
ERROR: No matching distribution found for kedro-docker
```

## Your Environment
Include as many relevant details as possible about the environment you experienced the bug in:

* Operating system and version: Ubuntu 20.04
* Kedro version used (if relevant): 0.16.1
* Python version used (if relevant): 3.8.2

**Comment by lorenabalan** [comment]
Hi @fkromer , thank you for raising. This has been addressed as of https://github.com/quantumblacklabs/kedro-docker/commit/163acc6d14ed01d20ffd01de4e458275828c8c89. It's available in `kedro-docker>=0.2.0`, so closing this as resolved.

**Comment by fkromer** [comment]
@lorenabalan Great. Thanks a lot.
